docs/providers/codex-cli.md

```
@@ -0,0 +1,506 @@
+}
+```
+
+## Troubleshooting
+
+### "codex: command not found" Error
+
+**Symptoms**: Task Master reports that the Codex CLI is not found.
+
+**Solutions**:
+1. **Install Codex CLI globally**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)